.github/workflows/pr-checks.yml

```diff
@@ -119,6 +119,9 @@ jobs:
   e2e-test:
     runs-on: ubuntu-latest
     needs: docker-local
+    strategy:
+      matrix:
+        node_image: [ v1.21, v1.22, v1.23, v1.24, v1.25, v1.26, v1.27 ]
```

Comment:
I have concerns with the breadth of this matrix, both due to resource consumption and supportability. 1.24 support ends in 3 days, I suggest we just do the oldest/newest supported, so reduce the matrix to `[v1.25, v1.27]`

Reply:
the goal is to give it a try to so which version is currently supported and fixeable.